### PR TITLE
Added toast library to replace current system

### PR DESCRIPTION
**Pull request by joshua-smee-ttd**
Added [React-Toastify](https://www.npmjs.com/package/react-toastify) to use to show popup notifications.

Is much easier to use then current system. Is currently only implemented in some API Key management functions but future aim would be to completely replace current `StatusPopup`.

chrome_3af5wGgh0i

### CODE REVIEW

File: src/web/styles/themes.scss [hunk 1]
```diff
@@ -28,7 +28,7 @@
   --theme-error-background: #facec5;
   --theme-success-background: #c0d5d5;
   --theme-warning-background: #fefcd3;
-  --theme-info-icon: #7085d4;
+  --theme-info: #7085d4;
```
Comment (joshua-smee-ttd) [inline src/web/styles/themes.scss:31]: Renamed colour to match pattern

File: src/web/styles/toaster.scss [hunk 2]
```diff
@@ -0,0 +1,35 @@
+.app {
```
Comment (alex-yau-ttd) [inline src/web/styles/toaster.scss:1]: nit: maybe call this file `toast.scss`

Comment (joshua-smee-ttd) [reply src/web/styles/toaster.scss:1]: Sounds good!

**Review by alex-yau-ttd — APPROVED** [review]

**Comment by alex-yau-ttd** [comment]
Is it possible we could align the styling more with the Figma? 
image

For example, the colouring & line-height could probably be easily changed
Old toast
image

New toast
image

Then possibly the "Info" heading, etc.

File: src/web/components/Core/Toast.stories.tsx [hunk 3]
```diff
@@ -0,0 +1,55 @@
+import { Meta, StoryObj } from '@storybook/react';
+
+import { ErrorToast, InfoToast, SuccessToast, ToastContainerWrapper, WarningToast } from './Toast';
+
+function TestToaster() {
```
Comment (alex-yau-ttd) [inline src/web/components/Core/Toast.stories.tsx:5]: `TestToast`?

**Review by alex-yau-ttd — APPROVED** [review]

**Pull request merged**